activesupport/CHANGELOG.md

```diff
@@ -114,4 +114,10 @@
 
     *fatkodima*
 
+*   Add a new optoin `:early_expiration` to `ActiveSupport::Cache::Store` to allow for probabilistic early expiration of cache entries.
```

Comment:
```suggestion
*   Add a new option `:early_expiration` to `ActiveSupport::Cache::Store` to allow for probabilistic early expiration of cache entries.
```

Reply:
Fixed the typo. Thanks for the reminder :)